### PR TITLE
Brazilian portuguese translation

### PR DESCRIPTION
**Pull request by vinipitta**
Full dictionary translation for Brazilian Portuguese. 

@lucaslf @morgade can help to review translation.

### CODE REVIEW

**Comment by alexcjohnson** [comment]
Thanks @vinipitta - looks great!

Our [syntax check](https://circleci.com/gh/plotly/plotly.js/9744?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) is complaining about the number of spaces for the translation keys (should be 4 more than the level above, not 8) and some trailing spaces at the end of lines and on blank lines.

I'm on the fence about the `kde` translation - most other translators so far have left that as the English abbreviation, because spelling it out completely could be a bit long for the tooltip where this appears:
screen shot 2018-05-09 at 9 45 19 am
But if you think people will be confused by `kde` by itself I'm happy to leave it as you have it.

**Comment by vinipitta** [comment]
@alexcjohnson,  I'd totally agree about _kde_ translation. I am not familiar with this type of plot so looked for academic publications about it to base my translation but I just find one reference. So I am comfortable in leave the original abbreviation because people who employ, very likely found it concept from English sources.

File: lib/locales/pt-br.js [hunk 1]
```diff
@@ -11,7 +11,66 @@
...
+        'Click to enter Colorscale title': 'Clique para editar o título da escala de cor',
+        'Click to enter Component A title': 'Clique para editar o título do Componente A',
+        'Click to enter Component B title': 'Clique para editar o título do Componente B',
+        'Click to enter Component C title': 'Clique para editar o título do Componente C',
+        'Click to enter Plot title': 'Clique para editar o título do Gráfico',
+        'Click to enter X axis title': 'Clique para editar o título do eixo X',
+        'Click to enter Y axis title': 'Clique para editar o título do eixo Y',
+        'Click to enter radial axis title': 'Clique para editar o título do eixo radial',
+        'Compare data on hover': 'Comparar dados ao pairar',
+        'Double-click on legend to isolate one trace': 'Duplo clique na legenda para isolar uma serie',
```
Comment (andrefarzat) [inline lib/locales/pt-br.js]: Falta acentuação em "s**é**rie" ?

File: lib/locales/pt-br.js [hunk 2]
```diff
@@ -11,7 +11,66 @@
...
+        'Double-click to zoom back out': 'Duplo clique para reverter zoom',
+        'Download plot as a png': 'Fazer download do gráfico como imagem (png)',
+        'Edit in Chart Studio': 'Editar no Chart Studio',
+        'IE only supports svg.  Changing format to svg.': 'IE suporta apenas svg. Alterando formato para svg',
+        'Lasso Select': 'Seleção de laço',
+        'Orbital rotation': 'Rotação orbital',
+        'Pan': 'Mover',
+        'Produced with Plotly': 'Produzido com o Plotly',
```
Comment (andrefarzat) [inline lib/locales/pt-br.js]: Talvez `Criado com Plotly` ?

File: lib/locales/pt-br.js [hunk 3]
```diff
@@ -11,7 +11,66 @@
...
+        'Reset camera to last save': 'Restaurar câmera para última salva',
+        'Reset view': 'Restaurar visão',
+        'Reset views': 'Restaurar visões',
+        'Show closest data on hover': 'Exibir dado mais próximo ao pairar',
+        'Snapshot succeeded': 'Captura instantânea completa',
+        'Sorry, there was a problem downloading your snapshot!': 'Desculpe, houve um problema no download de sua captura instantânea!',
+        'Taking snapshot - this may take a few seconds': 'Efetuando captura instantânea - isso pode levar alguns instantes',
+        'Toggle Spike Lines': 'Habilitar/desabilitar triangulação de linhas',
+        'Toggle show closest data on hover': 'Habilitar/desabilitar exibição de dado mais próximo ao pairar',
+        'Turntable rotation': 'Rotação de mesa',
```
Comment (andrefarzat) [inline lib/locales/pt-br.js]: Essa ficou confusa, mas não tenho sugestão melhor também :/

File: lib/locales/pt-br.js [hunk 4]
```diff
@@ -11,7 +11,66 @@
...
+        'Zoom': 'Zoom',
+        'Zoom in': 'Ampliar zoom',
+        'Zoom out': 'Reduzir zoom',
+        'close': 'fechamento',
```
Comment (andrefarzat) [inline lib/locales/pt-br.js]: não seria "fechar" ?

File: lib/locales/pt-br.js [hunk 5]
```diff
@@ -11,7 +11,66 @@
...
+        'lon': 'longitude',
+        'low': 'baixa',
+        'lower fence': 'limite inferior',
+        'max': 'máximo',
+        'mean ± σ': 'média ± σ',
+        'mean': 'média',
+        'median': 'mediana',
+        'min': 'mínimo',
+        'new text': 'novo texto',
+        'open': 'abertura',
```
Comment (andrefarzat) [inline lib/locales/pt-br.js]: não seria "abrir" ?

File: lib/locales/pt-br.js (hunk 1 above)
Comment (vinipitta) [reply lib/locales/pt-br.js]: Sim, farei a correção.

File: lib/locales/pt-br.js (hunk 3 above)
Comment (vinipitta) [reply lib/locales/pt-br.js]: Acredito que você esteja se referindo ao "hover". De fato não temos termo equivalente em português que seja popular e amplamente usado.

File: lib/locales/pt-br.js (hunk 4 above)
Comment (vinipitta) [reply lib/locales/pt-br.js]: Não. A tradução está no contexto do gráfico OHLC, usado na bolsa de valores: https://blog.bussoladoinvestidor.com.br/grafico-ohlc/

Do arquivo translation-keys.txt: // traces/ohlc/calc.js:104

File: lib/locales/pt-br.js (hunk 5 above)
Comment (vinipitta) [reply lib/locales/pt-br.js]: Não.  Se enquadra na mesma situação do fechamento.

Obrigado pelas contribuições! Farei as correções dos pontos em que concordamos.

**Comment by alexcjohnson** [comment]
Thanks again @vinipitta - very nice work!

**Pull request merged**